src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.html

```diff
@@ -1,14 +1,36 @@
-<ng-container *ngIf="shouldShowHoverScrub && !isMediaUnavailable">
-  <div #tooltipHoverElement class="c-hover-scrub-hover-element"></div>
-
-  <sofie-tooltip [tooltipElementHoverMousePosition]="tooltipElementHoverMousePosition">
-    <sofie-video-hover-scrub
-      [filename]="piece.name"
-      [playedDurationInMs]="playedDurationForPartInMs"
-      [tooltipHoverElementWidth]="tooltipHoverElement.clientWidth"
-      [tooltipElementHoverMousePosition]="tooltipElementHoverMousePosition"
-      [durationInMs]="durationInMs"
-      [isJingle]="tv2PieceType === Tv2PieceType.JINGLE">
-    </sofie-video-hover-scrub>
+  <div #tooltipHoverElement class="tooltip-hover-element"></div>
+
+  <sofie-tooltip *ngIf="tooltipContentFields.length > 0" [tooltipElementHoverMousePosition]="tooltipElementHoverMousePosition">
+    <div class="tooltip">
+      <sofie-video-hover-scrub *ngIf="shouldShowHoverScrub && !isMediaUnavailable"
```

Comment:
Just a note on the `isMediaUnavailable` variable. In general negated statements are harder to grasp, so we could consider if we should change it to `isMediaAvailable`.

Reply:
I've kept it as is for now, it seems to be used in cases where we want the media to be unavailable. I do see your point, however!